### PR TITLE
Fixes to accommodate storybook version upgrade

### PR DESCRIPTION
**Pull request by jtbandes**
cc @tsuyoshiwada
Fixes the build failure in #92 by applying various changes to support Storybook 4. A few notes:

- Added `--ci` so the storybook server doesn't open a browser automatically.
- Changed the regex being used to process the start-storybook output because their output format changed as noted in https://github.com/tsuyoshiwada/storybook-chrome-screenshot/issues/92#issuecomment-435468282
- Had to get rid of `require.context` in the test config files because I wasn't able to get `babel-plugin-require-context-hook` to work. Hopefully that wasn't super important :)

### CODE REVIEW

File: src/core/app/Server.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export class Server {
 
   private static matchServerURL(buffer: string | Buffer) {
     const str = buffer.toString().trim();
-    const m = str.match(/Storybook started on => (https?:\/\/.+)/);
+    const m = str.match(/(?:started(?:.|\n)+Local:\s+)(https?:\/\/\S+)/);
```
Comment (LucianBuzzo) [inline src/core/app/Server.ts]: If we wanted something with more specificity, you could try:
```
/\s+Storybook\s\S*\sstarted\s+\│\s+\│\s+\│\s+\│\s+Local:\s+(https?:\/\/.+)/gm)
```
Which would match the leading `Storybook` message and the following empty line

File: src/core/app/Server.ts [hunk 2]
```diff
@@ -47,7 +47,7 @@ export class Server {
 
   private static matchServerURL(buffer: string | Buffer) {
     const str = buffer.toString().trim();
-    const m = str.match(/Storybook started on => (https?:\/\/.+)/);
+    const m = str.match(/started(?:.|\n)+Local:\s+(https?:\/\/\S+)/);
```
Comment (Glavin001) [inline src/core/app/Server.ts]: Testing this by replacing locally within `node_modules/`. Thank you 🎉 . Looking forward to this being merged and published! Totally blocked with Storybook 4 right now.

**Comment by Glavin001** [comment]
Since I was blocked until this is merged, I temporarily published to https://www.npmjs.com/package/@review-packs/storybook-chrome-screenshot

If you're using `Yarn` you can use an [alias](https://twitter.com/sebmck/status/873958247304232961?s=20):

```diff
-    "storybook-chrome-screenshot": "1.3.1",
+    "storybook-chrome-screenshot": "npm:@review-packs/storybook-chrome-screenshot@latest",
```

**Comment by tsuyoshiwada** [comment]
Sorry for late... :bow:
Released as [v1.4.0](https://www.npmjs.com/package/storybook-chrome-screenshot/v/1.4.0) :tada:

**Comment by LucianBuzzo** [comment]
@tsuyoshiwada I think this should be a major version update, as its not backwards compatible AFAICT

**Pull request merged**